### ISSUE TITLE
Property:source

### ISSUE THREAD

**Issue by mtrekels** [tdwg/cd#128]
|  |  |
| ---- | ---- |
| **Definition** |  |
| **Dimension** |  |
| **Existing property** |  |
| **Existing class** |  |
| **Existing property identifier** |  |
| **Format** |  |
| **Required** |  |
| **Repeatable** |  |
| **Constraints** |  |
| **Examples** |  |
| **Notes** |  |

**Comment by mswoodburn** [comment]
Definition is currently the very generic one for dcterms:source, however I think we need one more specific to the collectionIdentifierSource context (with the understanding that this is a subproperty of dcterms:source)

**Comment by qgroom** [comment]
Yes, I agree and should it really be synonymous with dcterms:source?
